spec/models/form526_submission_spec.rb

```diff
@@ -1282,7 +1282,17 @@ def expect_no_max_cfi_logged(diagnostic_code)
       context 'with submission confirmation email when successful job statuses' do
         subject { create(:form526_submission, :with_multiple_succesful_jobs) }
 
-        it 'returns one job triggered' do
+        it 'does not trigger job when disability_526_call_received_email_from_polling enabled' do
+          allow(Flipper).to receive(:enabled?).with(:disability_526_call_received_email_from_polling,
+                                                    anything).and_return(true)
+          expect do
+            subject.workflow_complete_handler(nil, 'submission_id' => subject.id)
+          end.to change(Form526ConfirmationEmailJob.jobs, :size).by(0)
+        end
+
+        it 'returns one job triggered when disability_526_call_received_email_from_polling disabled' do
+          allow(Flipper).to receive(:enabled?).with(:disability_526_call_received_email_from_polling,
```

Comment:
here too!
```
allow(Flipper).to receive(:enabled?).with(:saved_claim_schema_validation_disable).and_return(false)
```

Reply:
hm, that is odd. I'll bring that up with the backend CoP